lnwire/accept_channel.go

```diff
@@ -281,3 +285,10 @@ func (a *AcceptChannel) Decode(r io.Reader, pver uint32) error {
 func (a *AcceptChannel) MsgType() MessageType {
 	return MsgAcceptChannel
 }
+
+// SerializedSize returns the serialized size of the message in bytes.
+//
+// This is part of the lnwire.SizeableMessage interface.
+func (a *AcceptChannel) SerializedSize() (uint32, error) {
+	return MessageSerializedSize(a)
```

Comment:
Q: should we buffer the calculated result? Was wondering about the impact on the performance, but I guess this method is only called when we check for rate limit so it should be fine?

Reply:
Yeah it's only called when we check the rate limit. There's def some more optimization we can do here (eg: we can cache the serialization, so we only ever compute it once). Right now in the gossip syncer code, we'll end up computing the encoding twice (once to check the limit, then again to send). 

I prefer to leave that for a future PR, as possible we run into some consistency issues along the way.